.github/scripts/tests/transform-ya-junit.py

```diff
@@ -43,10 +43,41 @@ def populate(self, testsuite, testcase):
 
         self.regexps.append(tuple(check))
 
-    def __call__(self, suite_name, test_name):
+    def __call__(self, suite_name, test_name, test_case):
```

Comment:
Could you please attach an example of these values?

Reply:
test_case is a xml etree object correlating to:
```
<testcase name="[11/16] chunk chunk" time="0.0" classname="cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests">
            <failure>Chunk exceeded timeout - dumping goroutine stacktraces
                PC=0x10ab90e m=0 sigcode=0

                goroutine 0 [idle]:
                runtime/internal/syscall.Syscall6()
                /-S/contrib/go/_std_1.21/src/runtime/internal/syscall/asm_linux_amd64.s:36 +0xe fp=0x7ffe2cace010 sp=0x7ffe2cace008 pc=0x10ab90e
                runtime/internal/syscall.EpollWait(0x5?, {0x7ffe2cace0d8?, 0x7ffe00000010?, 0x7ffe00000001?}, 0xa2?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/internal/syscall/syscall_linux.go:55 +0x45 fp=0x7ffe2cace060 sp=0x7ffe2cace010 pc=0x10ab885
                runtime.netpoll(0xc00005e000?)
                /-S/contrib/go/_std_1.21/src/runtime/netpoll_epoll.go:118 +0xcb fp=0x7ffe2cace6f0 sp=0x7ffe2cace060 pc=0x10dc6eb
                runtime.findRunnable()
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:3191 +0x825 fp=0x7ffe2cace800 sp=0x7ffe2cace6f0 pc=0x10e8945
                runtime.schedule()
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:3582 +0xb1 fp=0x7ffe2cace838 sp=0x7ffe2cace800 pc=0x10e9ab1
                runtime.park_m(0xc0001824e0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:3745 +0x11f fp=0x7ffe2cace880 sp=0x7ffe2cace838 pc=0x10e9fbf
                traceback: unexpected SPWRITE function runtime.mcall
                runtime.mcall()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:458 +0x4e fp=0x7ffe2cace898 sp=0x7ffe2cace880 pc=0x11148ae

                goroutine 1 [chan receive, 9 minutes]:
                runtime.gopark(0xc00071f9e8?, 0x10b5da5?, 0x10?, 0x25?, 0x18?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000033980 sp=0xc000033960 pc=0x10e35ee
                runtime.chanrecv(0xc00004c2a0, 0xc00071fa67, 0x1)
                /-S/contrib/go/_std_1.21/src/runtime/chan.go:583 +0x3cd fp=0xc0000339f8 sp=0xc000033980 pc=0x10af12d
                runtime.chanrecv1(0x1cf65e0?, 0x29fca0?)
                /-S/contrib/go/_std_1.21/src/runtime/chan.go:442 +0x12 fp=0xc000033a20 sp=0xc0000339f8 pc=0x10aed32
                testing.(*T).Run(0xc000007d40, {0x516c68?, 0x11cd85c?}, 0x5516f0)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1649 +0x3c8 fp=0xc000033ae0 sp=0xc000033a20 pc=0x11ceda8
                testing.runTests.func1(0x1cf8d20?)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:2054 +0x3e fp=0xc000033b30 sp=0xc000033ae0 pc=0x11d0ebe
                testing.tRunner(0xc000007d40, 0xc00071fc48)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1595 +0xff fp=0xc000033b80 sp=0xc000033b30 pc=0x11cdf7f
                testing.runTests(0xc000221360?, {0x1c31da0, 0x17, 0x17}, {0xc?, 0xc00071fd08?, 0x0?})
                /-S/contrib/go/_std_1.21/src/testing/testing.go:2052 +0x445 fp=0xc000033c78 sp=0xc000033b80 pc=0x11d0da5
                testing.(*M).Run(0xc000221360)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1925 +0x636 fp=0xc000033ec0 sp=0xc000033c78 pc=0x11cf796
                main.main()
                /-B/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/_test_main.go:49 +0x19c fp=0xc000033f40 sp=0xc000033ec0 pc=0x1b4859c
                runtime.main()
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:267 +0x2bb fp=0xc000033fe0 sp=0xc000033f40 pc=0x10e317b
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000033fe8 sp=0xc000033fe0 pc=0x11168a1

                goroutine 2 [force gc (idle), 3 minutes]:
                runtime.gopark(0x15a79eab359?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000f0fa8 sp=0xc0000f0f88 pc=0x10e35ee
                runtime.goparkunlock(...)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:404
                runtime.forcegchelper()
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:322 +0xb3 fp=0xc0000f0fe0 sp=0xc0000f0fa8 pc=0x10e3453
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000f0fe8 sp=0xc0000f0fe0 pc=0x11168a1
                created by runtime.init.6 in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:310 +0x1a

                goroutine 18 [GC sweep wait]:
                runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000ec778 sp=0xc0000ec758 pc=0x10e35ee
                runtime.goparkunlock(...)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:404
                runtime.bgsweep(0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/mgcsweep.go:321 +0xdf fp=0xc0000ec7c8 sp=0xc0000ec778 pc=0x10cd8df
                runtime.gcenable.func1()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:200 +0x25 fp=0xc0000ec7e0 sp=0xc0000ec7c8 pc=0x10c2a25
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000ec7e8 sp=0xc0000ec7e0 pc=0x11168a1
                created by runtime.gcenable in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:200 +0x66

                goroutine 19 [GC scavenge wait]:
                runtime.gopark(0x3c5186ae?, 0x3b9aca00?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000ecf70 sp=0xc0000ecf50 pc=0x10e35ee
                runtime.goparkunlock(...)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:404
                runtime.(*scavengerState).park(0x1cf8120)
                /-S/contrib/go/_std_1.21/src/runtime/mgcscavenge.go:425 +0x49 fp=0xc0000ecfa0 sp=0xc0000ecf70 pc=0x10cb129
                runtime.bgscavenge(0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/mgcscavenge.go:658 +0x59 fp=0xc0000ecfc8 sp=0xc0000ecfa0 pc=0x10cb6d9
                runtime.gcenable.func2()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:201 +0x25 fp=0xc0000ecfe0 sp=0xc0000ecfc8 pc=0x10c29c5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000ecfe8 sp=0xc0000ecfe0 pc=0x11168a1
                created by runtime.gcenable in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:201 +0xa5

                goroutine 34 [finalizer wait, 9 minutes]:
                runtime.gopark(0x4df500?, 0x1010e4701?, 0x0?, 0x0?, 0x10eb7a5?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000f0628 sp=0xc0000f0608 pc=0x10e35ee
                runtime.runfinq()
                /-S/contrib/go/_std_1.21/src/runtime/mfinal.go:193 +0x107 fp=0xc0000f07e0 sp=0xc0000f0628 pc=0x10c1aa7
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000f07e8 sp=0xc0000f07e0 pc=0x11168a1
                created by runtime.createfing in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mfinal.go:163 +0x3d

                goroutine 20 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000ed750 sp=0xc0000ed730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0000ed7e0 sp=0xc0000ed750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000ed7e8 sp=0xc0000ed7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 50 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000586750 sp=0xc000586730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005867e0 sp=0xc000586750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005867e8 sp=0xc0005867e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 51 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000586f50 sp=0xc000586f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000586fe0 sp=0xc000586f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000586fe8 sp=0xc000586fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 52 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000587750 sp=0xc000587730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005877e0 sp=0xc000587750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005877e8 sp=0xc0005877e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 53 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000587f50 sp=0xc000587f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000587fe0 sp=0xc000587f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000587fe8 sp=0xc000587fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 54 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000588750 sp=0xc000588730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005887e0 sp=0xc000588750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005887e8 sp=0xc0005887e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 55 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000588f50 sp=0xc000588f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000588fe0 sp=0xc000588f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000588fe8 sp=0xc000588fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 56 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000589750 sp=0xc000589730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005897e0 sp=0xc000589750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005897e8 sp=0xc0005897e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 57 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000589f50 sp=0xc000589f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000589fe0 sp=0xc000589f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000589fe8 sp=0xc000589fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 58 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000582750 sp=0xc000582730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005827e0 sp=0xc000582750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005827e8 sp=0xc0005827e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 59 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000582f50 sp=0xc000582f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000582fe0 sp=0xc000582f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000582fe8 sp=0xc000582fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 60 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000583750 sp=0xc000583730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005837e0 sp=0xc000583750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005837e8 sp=0xc0005837e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 61 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000583f50 sp=0xc000583f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000583fe0 sp=0xc000583f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000583fe8 sp=0xc000583fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 62 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000584750 sp=0xc000584730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005847e0 sp=0xc000584750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005847e8 sp=0xc0005847e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 63 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000584f50 sp=0xc000584f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000584fe0 sp=0xc000584f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000584fe8 sp=0xc000584fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 64 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000585750 sp=0xc000585730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005857e0 sp=0xc000585750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005857e8 sp=0xc0005857e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 65 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000585f50 sp=0xc000585f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000585fe0 sp=0xc000585f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000585fe8 sp=0xc000585fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 66 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000594750 sp=0xc000594730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005947e0 sp=0xc000594750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005947e8 sp=0xc0005947e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 67 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000594f50 sp=0xc000594f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000594fe0 sp=0xc000594f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000594fe8 sp=0xc000594fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 68 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000595750 sp=0xc000595730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005957e0 sp=0xc000595750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005957e8 sp=0xc0005957e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 69 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000595f50 sp=0xc000595f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000595fe0 sp=0xc000595f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000595fe8 sp=0xc000595fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 70 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000596750 sp=0xc000596730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005967e0 sp=0xc000596750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005967e8 sp=0xc0005967e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 71 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000596f50 sp=0xc000596f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000596fe0 sp=0xc000596f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000596fe8 sp=0xc000596fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 72 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000597750 sp=0xc000597730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005977e0 sp=0xc000597750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005977e8 sp=0xc0005977e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 73 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000597f50 sp=0xc000597f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000597fe0 sp=0xc000597f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000597fe8 sp=0xc000597fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 74 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000590750 sp=0xc000590730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005907e0 sp=0xc000590750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005907e8 sp=0xc0005907e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 3 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000f1750 sp=0xc0000f1730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0000f17e0 sp=0xc0000f1750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000f17e8 sp=0xc0000f17e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 75 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000590f50 sp=0xc000590f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000590fe0 sp=0xc000590f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000590fe8 sp=0xc000590fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 21 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000edf50 sp=0xc0000edf30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0000edfe0 sp=0xc0000edf50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000edfe8 sp=0xc0000edfe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 22 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000ee750 sp=0xc0000ee730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0000ee7e0 sp=0xc0000ee750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000ee7e8 sp=0xc0000ee7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 23 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0000eef50 sp=0xc0000eef30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0000eefe0 sp=0xc0000eef50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000eefe8 sp=0xc0000eefe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 76 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000591750 sp=0xc000591730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005917e0 sp=0xc000591750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005917e8 sp=0xc0005917e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 77 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000591f50 sp=0xc000591f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000591fe0 sp=0xc000591f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000591fe8 sp=0xc000591fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 78 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000592750 sp=0xc000592730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005927e0 sp=0xc000592750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005927e8 sp=0xc0005927e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 79 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000592f50 sp=0xc000592f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000592fe0 sp=0xc000592f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000592fe8 sp=0xc000592fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 80 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000593750 sp=0xc000593730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005937e0 sp=0xc000593750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005937e8 sp=0xc0005937e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 81 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000593f50 sp=0xc000593f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc000593fe0 sp=0xc000593f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000593fe8 sp=0xc000593fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 82 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059e750 sp=0xc00059e730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059e7e0 sp=0xc00059e750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059e7e8 sp=0xc00059e7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 83 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059ef50 sp=0xc00059ef30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059efe0 sp=0xc00059ef50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059efe8 sp=0xc00059efe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 84 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059f750 sp=0xc00059f730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059f7e0 sp=0xc00059f750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059f7e8 sp=0xc00059f7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 85 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059ff50 sp=0xc00059ff30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059ffe0 sp=0xc00059ff50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059ffe8 sp=0xc00059ffe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 86 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a0750 sp=0xc0005a0730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a07e0 sp=0xc0005a0750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a07e8 sp=0xc0005a07e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 87 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a0f50 sp=0xc0005a0f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a0fe0 sp=0xc0005a0f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a0fe8 sp=0xc0005a0fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 88 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a1750 sp=0xc0005a1730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a17e0 sp=0xc0005a1750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a17e8 sp=0xc0005a17e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 89 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a1f50 sp=0xc0005a1f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a1fe0 sp=0xc0005a1f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a1fe8 sp=0xc0005a1fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 90 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059a750 sp=0xc00059a730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059a7e0 sp=0xc00059a750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059a7e8 sp=0xc00059a7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 91 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059af50 sp=0xc00059af30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059afe0 sp=0xc00059af50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059afe8 sp=0xc00059afe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 92 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059b750 sp=0xc00059b730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059b7e0 sp=0xc00059b750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059b7e8 sp=0xc00059b7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 93 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059bf50 sp=0xc00059bf30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059bfe0 sp=0xc00059bf50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059bfe8 sp=0xc00059bfe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 94 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059c750 sp=0xc00059c730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059c7e0 sp=0xc00059c750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059c7e8 sp=0xc00059c7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 95 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059cf50 sp=0xc00059cf30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059cfe0 sp=0xc00059cf50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059cfe8 sp=0xc00059cfe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 96 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059d750 sp=0xc00059d730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059d7e0 sp=0xc00059d750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059d7e8 sp=0xc00059d7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 97 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc00059df50 sp=0xc00059df30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc00059dfe0 sp=0xc00059df50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00059dfe8 sp=0xc00059dfe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 98 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a8750 sp=0xc0005a8730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a87e0 sp=0xc0005a8750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a87e8 sp=0xc0005a87e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 99 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a8f50 sp=0xc0005a8f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a8fe0 sp=0xc0005a8f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a8fe8 sp=0xc0005a8fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 100 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a9750 sp=0xc0005a9730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a97e0 sp=0xc0005a9750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a97e8 sp=0xc0005a97e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 101 [GC worker (idle), 9 minutes]:
                runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005a9f50 sp=0xc0005a9f30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005a9fe0 sp=0xc0005a9f50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005a9fe8 sp=0xc0005a9fe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 102 [GC worker (idle)]:
                runtime.gopark(0x15a7a2a1ff1?, 0x1?, 0x2e?, 0x57?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005aa750 sp=0xc0005aa730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005aa7e0 sp=0xc0005aa750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005aa7e8 sp=0xc0005aa7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 103 [GC worker (idle), 3 minutes]:
                runtime.gopark(0x15a7a2a2ae2?, 0x0?, 0x0?, 0x0?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005aaf50 sp=0xc0005aaf30 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005aafe0 sp=0xc0005aaf50 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005aafe8 sp=0xc0005aafe0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 104 [GC worker (idle)]:
                runtime.gopark(0x1d2a520?, 0x1?, 0x89?, 0x88?, 0x0?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005ab750 sp=0xc0005ab730 pc=0x10e35ee
                runtime.gcBgMarkWorker()
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1293 +0xe5 fp=0xc0005ab7e0 sp=0xc0005ab750 pc=0x10c45a5
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005ab7e8 sp=0xc0005ab7e0 pc=0x11168a1
                created by runtime.gcBgMarkStartWorkers in goroutine 1
                /-S/contrib/go/_std_1.21/src/runtime/mgc.go:1217 +0x1c

                goroutine 24 [chan receive, 9 minutes]:
                runtime.gopark(0xc000101df8?, 0x10b5da5?, 0x58?, 0x25?, 0x18?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc000101d90 sp=0xc000101d70 pc=0x10e35ee
                runtime.chanrecv(0xc00004c310, 0xc000101e77, 0x1)
                /-S/contrib/go/_std_1.21/src/runtime/chan.go:583 +0x3cd fp=0xc000101e08 sp=0xc000101d90 pc=0x10af12d
                runtime.chanrecv1(0x1cf65e0?, 0x29fca0?)
                /-S/contrib/go/_std_1.21/src/runtime/chan.go:442 +0x12 fp=0xc000101e30 sp=0xc000101e08 pc=0x10aed32
                testing.(*T).Run(0xc000202b60, {0x4f6924?, 0x5ad?}, 0xc00005d5d0)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1649 +0x3c8 fp=0xc000101ef0 sp=0xc000101e30 pc=0x11ceda8
                github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage.TestShallowCopySnapshotWithRandomFailure(0x0?)
                /-S/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/storage_ydb_test.go:764 +0x8c fp=0xc000101f70 sp=0xc000101ef0 pc=0x1b3f96c
                testing.tRunner(0xc000202b60, 0x5516f0)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1595 +0xff fp=0xc000101fc0 sp=0xc000101f70 pc=0x11cdf7f
                testing.(*T).Run.func1()
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1648 +0x25 fp=0xc000101fe0 sp=0xc000101fc0 pc=0x11cef05
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000101fe8 sp=0xc000101fe0 pc=0x11168a1
                created by testing.(*T).Run in goroutine 1
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1648 +0x3ad

                goroutine 25 [sleep, 9 minutes]:
                runtime.gopark(0xea75548f2f?, 0xe6a050?, 0xa0?, 0xa2?, 0xe6a050?)
                /-S/contrib/go/_std_1.21/src/runtime/proc.go:398 +0xce fp=0xc0005abd70 sp=0xc0005abd50 pc=0x10e35ee
                time.Sleep(0x8bb2c97000)
                /-S/contrib/go/_std_1.21/src/runtime/time.go:195 +0x125 fp=0xc0005abdb0 sp=0xc0005abd70 pc=0x1113865
                github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage.TestShallowCopySnapshotWithRandomFailure.func1(0xc000202ea0)
                /-S/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/storage_ydb_test.go:765 +0x4f fp=0xc0005abf70 sp=0xc0005abdb0 pc=0x1b3fa6f
                testing.tRunner(0xc000202ea0, 0xc00005d5d0)
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1595 +0xff fp=0xc0005abfc0 sp=0xc0005abf70 pc=0x11cdf7f
                testing.(*T).Run.func1()
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1648 +0x25 fp=0xc0005abfe0 sp=0xc0005abfc0 pc=0x11cef05
                runtime.goexit()
                /-S/contrib/go/_std_1.21/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005abfe8 sp=0xc0005abfe0 pc=0x11168a1
                created by testing.(*T).Run in goroutine 24
                /-S/contrib/go/_std_1.21/src/testing/testing.go:1648 +0x3ad
                Chunk exceeded 600s timeout and was killed
                List of the tests involved in the launch:
                tests::TestShallowCopySnapshotWithRandomFailure (timeout) duration: 599.24s
                tests::TestShallowCopySnapshotWithRandomFailure/store_chunks_in_ydb test was not launched inside chunk.

                logsdir:
                /home/github/tmp/out/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff
                backtrace:
                /home/github/tmp/out/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff/backtrace.txt
                stderr:
                /home/github/tmp/out/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff/stderr
                log:
                /home/github/tmp/out/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/run_test.log</failure>
            <properties><property name="url:backtrace" value="https://github-actions-s3.website.nemax.nebius.cloud/ydb-platform/nbs/PR-check/7935491215/1/nebius-x86-64/logs/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff/backtrace.txt" /><property name="url:stderr" value="https://github-actions-s3.website.nemax.nebius.cloud/ydb-platform/nbs/PR-check/7935491215/1/nebius-x86-64/logs/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff/stderr" /><property name="url:log" value="https://github-actions-s3.website.nemax.nebius.cloud/ydb-platform/nbs/PR-check/7935491215/1/nebius-x86-64/logs/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/run_test.log" /></properties></testcase>
        <testcase name="tests.TestShallowCopySnapshotWithRandomFailure" time="599.2379372119904" classname="cloud/disk_manager/internal/pkg/dataplane/snapshot/storage
		/tests">
            <skipped>Killed by timeout (600 s)

                logsdir:
                /home/github/tmp/out/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff
                log:
                /home/github/tmp/out/cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests/test-results/tests/chunk11/testing_out_stuff/tests.TestShallowCopySnapshotWithRandomFailure.log</skipped><properties><property name="mute" value="automatically muted based on rules" /></properties></testcase>
        <testcase name="tests.TestShallowCopySnapshotWithRandomFailure/store_chunks_in_ydb" time="0.0" classname="cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/tests">
            <skipped>Test was not launched, because other tests have used all allotted time (600 s)</skipped>
        </testcase>
```

Reply:
For a simpler debugging here's the script, which tests the script which tests the mute check and the xml for to run it against:
```py

import sys
from xml.etree import ElementTree as ET
import importlib.util


spec = importlib.util.spec_from_file_location(
    "ya_junit",
    "transform-ya-junit.py",
)
ya_junit_module = importlib.util.module_from_spec(spec)
sys.modules["ya_junit"] = ya_junit_module
spec.loader.exec_module(ya_junit_module)

tree = ET.parse(sys.argv[1])
root = tree.getroot()
mute = ya_junit_module.YaMuteCheck()
mute.load(sys.argv[1])
for suite in root.findall("testsuite"):
    suite_name = suite.get("name")

    for case in suite.findall("testcase"):
        test_name = case.get("name")
        if mute(suite_name, test_name, case):
            print(suite_name, test_name)

```
[ya-junit.txt](https://github.com/ydb-platform/nbs/files/14331950/ya-junit.txt)